CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## [v1.5.0](https://github.com/megaease/easegress/tree/v1.5.0) (2022-03-03)
+
+[Full Changelog](https://github.com/megaease/easegress/compare/v1.4.1...v1.5.0)
+
```

Comment:
Won't we deprecate the dynamic cluster and warn to delete it in the next release?

Reply:
Oh, forgot this, will update now.

Reply:
updated.